.github/workflows/main.yml

```diff
@@ -1,18 +1,48 @@
 name: CI
-on: push
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+
+      - uses: actions/setup-node@v3
         with:
-          node-version: '15'
```

Comment:
This is changed to fix even number versioning and for LTS, because odd number versioning of node is experimental and generally for development.